sdk/devopsinfrastructure/azure-resourcemanager-devopsinfrastructure/src/main/java/com/azure/resourcemanager/devopsinfrastructure/models/ManagedServiceIdentity.java

```diff
@@ -9,33 +9,32 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Map;
-import java.util.UUID;
 
 /**
- * Managed service identity (system assigned and/or user assigned identities).
+ * The properties of the managed service identities assigned to this resource.
  */
 @Fluent
 public final class ManagedServiceIdentity {
     /*
-     * The service principal ID of the system assigned identity. This property will only be provided for a system assigned identity.
+     * The Active Directory tenant id of the principal.
      */
-    @JsonProperty(value = "principalId", access = JsonProperty.Access.WRITE_ONLY)
-    private UUID principalId;
+    @JsonProperty(value = "tenantId", access = JsonProperty.Access.WRITE_ONLY)
+    private String tenantId;
```

Comment:
For mgmt plane, UUID should be generated as UUID.

Reply:
What's usually in our premium modules? String or UUID?

Personally I do not care much, as long as they are mostly consistent across module (but unlikely as v3 and v5 is different).

Reply:
It's UUID in premium: https://github.com/XiaofeiCao/azure-sdk-for-java/blob/872abd22baa353adca7d08df1b59d7c707e8f225/sdk/apicenter/azure-resourcemanager-apicenter/src/main/java/com/azure/resourcemanager/apicenter/models/ManagedServiceIdentity.java#L19-L31

I double checked, Swagger v3 and v5 are all UUID...
v3: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/managedidentity.json#L53-L64
v5: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json#L21-L34